vnext/metro.config.js

```diff
@@ -20,28 +15,22 @@ module.exports = {
   ],
 
   resolver: {
+    resolveRequest: require('./metro-react-native-platform').reactNativePlatformResolver(
+      {
+        windesktop: 'react-native-windows',
+        windows: 'react-native-windows',
+      },
+    ),
     extraNodeModules: {
-      // Redirect react-native and react-native-windows to this folder
-      'react-native': rnwPath,
+      // Redirect react-native-windows to this folder
       'react-native-windows': rnwPath,
     },
-    // Include the macos platform in addition to the defaults because the fork includes macos, but doesn't declare it
-    platforms: ['ios', 'android', 'windesktop', 'windows', 'web', 'macos'],
     // Since there are multiple copies of react-native, we need to ensure that metro only sees one of them
     // This should go away after RN 0.61 when haste is removed
```

Comment:
This comment can be removed from everywhere now.

Reply:
yup